### DISCUSSION TITLE
Mulighet til å legge til flere eiere av en tavle

### DISCUSSION THREAD

**Discussion by PeterRydberg** [entur/tavla#313] [Ideas]
Funksjonaliteten for å ha flere brukere i `owners` ligger strengt tatt inne allerede, men det er ingen mulighet på Tavla for å legge dem til. Jeg tenker dette kan gjøres forholdsvis lett gjennom adminpanelet og "Mine tavler". På "Mine tavler" kan brukeren ha en knapp som vedkommende trykker på, som åpner en modal med et inputfelt der man kan legge til noens email (e.l.). På adminpanelet kan man ha en liste over alle brukere i `owners`, som enten viser navn eller email avhengig av logintypen (Facebook, Google, email, etc.). Derfra kan man også legge til nye eiere i et inputfelt. Det kunne også vært praktisk med en liten indikasjon på om en tavle er delt i "Mine tavler", f.eks. et lite ikon ved tittelen.

**Comment by magnusrand** [comment]
Hei, Peter! Da kan jeg endelig meddele at i [v6.3.0](https://github.com/entur/tavla/releases/tag/v6.3.0) er nå funksjonaliteten for å legge til flere eiere av en tavle implementert!🤩 Da er bare å ha det morro med denne nye featuren!

**Reply by PeterRydberg** [reply]
Hallo, @magnusrand! Jeg fryder med overstadig over denne nye featuren 🤩 Faktisk fryder jeg meg så mye at jeg har valgt å dele en helt flunkende ny tavle med bare deg! @Collatty får så absolutt ikke være med.

Kudos for vanvittig bra innsats til @Collatty, @magnusrand, and @dependabot!

**Reply by Collatty** [reply]
![](https://media.giphy.com/media/BEob5qwFkSJ7G/giphy.gif)

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
